search/intermediate_result.cpp

```diff
@@ -279,20 +279,22 @@ void FillDetails(FeatureType & ft, Result::Details & details)
       }
     }
   }
-
-  details.m_isHotel = ftypes::IsHotelChecker::Instance()(ft);
+  
+  auto types_holder = feature::TypesHolder(ft);
```

Comment:
we use camelCase for variable names
`typesHolder`

Reply:
Fixed